app/code/Magento/Ui/view/base/web/js/grid/columns/select.js

```diff
@@ -21,10 +21,15 @@ define([
         getLabel: function () {
             var options = this.options || [],
                 values = this._super(),
+                multiSelectValues = [],
                 label = [];
 
             if (_.isString(values)) {
-                values = values.split(',');
+                multiSelectValues = values.split(',');
+
+                multiSelectValues = multiSelectValues.map(function (value) {
```

Comment:
Can you explain what you try to do here? It's only for converting values to a string?